chapters/en/slides/module7/module7_00.md

```diff
@@ -8,21 +8,23 @@ type: slides
 
 ## Module Learning Outcomes
 
-<br>
-
-Hi, and welcome to Module 7 of this course 👋
+By the end of this module, you will be able to
 
-<br>
+1. Explain what JupyterLab is.
+2. Install and use JupyterLab.
```

Comment:
Installing the software is not one of the learning objectives of module 0?

Reply:
Yes you are correct, it is still in the slides, but we don't need to highlight it as a learning objective. I will add a note about conda environments instead: "Work with Conda virtual environments inside JupyterLab"